addons/sale/tests/__init__.py

```diff
@@ -10,7 +10,6 @@
 from . import test_payment_flow
 from . import test_product_attribute_value
 from . import test_product_configurator_data
-from . import test_reinvoice
```

Comment:
removed from the init but not from the files? :thinking:

Reply:
We moved this test to sale_project module :)

Reply:
Probably didn't see it because github doesn't show it in the `sale` module diff, ok then :ok_hand: